client/app/libs/slim_lodash.js

```diff
@@ -0,0 +1,3 @@
+import bindAll from 'lodash/bindAll';
+import find from 'lodash/find';
+export default { bindAll, find };
```

Comment:
I agree. Does this syntax seem correct to you?

```javascript
import { bindAll, find } from 'lodash';
export default { bindAll, find };
```

This seems correct to me, and works fine on the dev build, but breaks the production build on this repo: `Uncaught ReferenceError: bindAll is not defined`. I unfortunately don't have enough experience with these technologies to be able to determine why.